CHANGELOG.md

```diff
@@ -9,8 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- _Upgrade (experimental)_: Migrate `grid-cols-[subgrid]` and `grid-rows-[subgrid]` to `grid-cols-subgrid` and `grid-rows-subgrid` ([#14840](https://github.com/tailwindlabs/tailwindcss/pull/14840))
+- _Upgrade (experimental)_: Rename `grid-cols-[subgrid]` and `grid-rows-[subgrid]` to `grid-cols-subgrid` and `grid-rows-subgrid` ([#14840](https://github.com/tailwindlabs/tailwindcss/pull/14840))
 - _Upgrade (experimental)_: Support migrating projects with multiple config files ([#14863](https://github.com/tailwindlabs/tailwindcss/pull/14863))
+- _Upgrade (experimental)_: Rename default utilities (such `shadow`, `blur` and `rounded`) to have a value prefix ([#14875](https://github.com/tailwindlabs/tailwindcss/pull/14875))
```

Comment:
"to have a value prefix" isn't clear (also I think you mean suffix here?)

Maybe we can reference the changed section in the changelog here somehow (.. rename `shadow`, `inset-shadow`, `drop-shadow`, `rounded`, and `blur` utilities in accordance with the changes made in this version)  it's clearer to what we do (or even repeat the exact examples that we have for `drop-shdow` to `drop-shadow-sm` etc.

Reply:
Yeah I think this is the clearest, albeit a bit verbose 👍

Reply:
That's exactly what we did in the PR where we introduced these changes, so just used that as a reference. https://github.com/tailwindlabs/tailwindcss/pull/14849/commits/d1724fdb06020f00a908dcb6d57580ee8756f867